odc/stac/_model.py

```diff
@@ -185,14 +182,32 @@ class RasterLoadParams:
     present and use this value instead.
     """
 
-    def __post_init__(self) -> None:
-        meta = self.src.meta
+    use_overviews: bool = True
+    """
+    Disable use of overview images.
+
+    Set to ``False`` to always read from the main image ignoring overview images
+    even present in the data source.
+    """
+
+    resampling: str = "nearest"
+    """Resampling method to use."""
```

Comment:
Could/should this be an enum?

Reply:
it gets converted eventually to this one:

https://rasterio.readthedocs.io/en/latest/api/rasterio.enums.html#rasterio.enums.Resampling

I guess it could be that. I certainly want to support string representation at least at construction time.